packages/palette/src/svgs/GuaranteeWhiteIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { Icon, IconProps, Path, Title } from "./Icon"
+
+/** GuaranteeWhiteIcon */
+export const GuaranteeWhiteIcon: React.FC<IconProps> = ({
+  title = "Guarantee White",
+  ...props
+}) => {
+  return (
+    <Icon {...props} viewBox="0 0 18 18">
+      <Title>{title}</Title>
+      <Path
+        fill="#fff"
```

Comment:
see other icons for example. should get rid of the other version too

Reply:
You mean `color(props.fill!)` , it says `color` is deprecated? Also the other one has 2 paths of 2 different colors.

Reply:
@dzucconi I've updated it to use one Guarantee icon with the color props.

Reply:
Thanks!